scripts/testnet/deploy_lynx.py

```diff
@@ -61,28 +51,30 @@ def main():
         publish=PUBLISH,
     )
 
-    # Lynx Ritual Token
     LynxRitualToken = deployer.deploy(
-        project.LynxRitualToken, Web3.to_wei(10_000_000, "ether"), publish=PUBLISH
+        *config.get_params(project.LynxRitualToken, locals()),
+        publish=PUBLISH
     )
 
     # Lynx Coordinator
     Coordinator = deployer.deploy(
-        project.Coordinator,  # coordinator
-        LynxTACoChildApplication.address,  # root_app
-        3600,  # timeout (seconds)
-        4,  # max_dkg_size
-        deployer.address,  # admin
-        LynxRitualToken.address,  # currency
-        1,  # fee_rate (wei per second)
+        *config.get_params(project.Coordinator, locals()),
         publish=PUBLISH,
     )
 
-    LynxTACoChildApplication.setCoordinator(Coordinator.address, sender=deployer)
+    LynxTACoChildApplication.setCoordinator(
+        Coordinator.address,
+        sender=deployer
+    )
 
-    # list deployments
-    deployments = [LynxRootApplication, LynxTACoChildApplication, LynxRitualToken, Coordinator]
+    deployments = [
+        LynxRootApplication,
+        LynxTACoChildApplication,
+        LynxRitualToken,
+        Coordinator
```

Comment:
I'd go with lowercase variables here, since these are contract instances, not contract containers (in ape parlance 🦍 ). When I read this initially I got confused.

Reply:
As it stands currently, these names match the variables in `lynx_config.json`.  Alternatively we could use lowercase names there too 🤔

Reply:
Yeah. We can change it with some additional logic, but for now it is named this way because we use `locals()` for the `context`.

Reply:
✔️